lib/import_school_group_data.rb

```diff
@@ -58,15 +58,18 @@ def save_csv_file(url, location)
   def convert_to_school_group(row)
     school_group = SchoolGroup.find_or_initialize_by(uid: row['Group UID'])
     set_gias_data_as_json(school_group, row)
-    set_geolocation(school_group, row)
+    school_group.postcode = row['Group Postcode']
+    school_group.name = row['Group Name']&.titlecase
+    school_group.address = row['Group Locality']
+    school_group.town = row['Group Town']
+    school_group.county = row['Group County']
+    set_geolocation(school_group)
     school_group
   end
 
-  def set_geolocation(school_group, row)
-    if row['Group Postcode']&.present? && (school_group.geolocation.blank? || school_group.postcode != row['Group Postcode'])
```

Comment:
This was intended to prevent unnecessary API usage and us running out of operations